src/models/state.ts

```diff
@@ -4,13 +4,16 @@ export interface AppState {
 	cache: {
 		matches: ICachedMatch[]
 	};
-	teleop: {
-		grid: number[];
-		chargeStation: number;
-	};
-	auto: {
-		grid: number[];
-		park: number;
-		chargeStation: number;
-	};
+	notes: {
+		"Auto": string, 
```

Comment:
These property names should not be strings; when property names are strings like this, you are forced to use `notes['property name']` to access the value, instead of dot notation `notes.propertyName`. This removes your IDE's ability to look for bugs, if something is misspelled or a name gets changed.

Reply:
I could see potential issues with this.
a) Some of the categories have multiple words which makes strings necessary in order to assign dispatches correctly. 
b) We are sending strings in our dispatch, which may need to be read as such in the switch case in Actions.js, but this can be easily fixed. We still run into the first issue though. 

The one word categories are fine and can be changed without issue, but the multiple word categories have a number of potential solutions:
1. Change everything to be 1 word - easy, but can impede use.
2. Make references in `QualitativePage.jsx` to assign 1 word variables to 2 word categories, and change action and reducer respectively - probably the best course of action. 
3. Leave as is - not a good idea.

Reply:
The original goal was to differentiate between the different actions by reading the `buttonText` variable, which was a string. Unsure if this matters, so will do some testing

Reply:
It looks to me like these are never accessed dynamically, eg `state.notes[someVariable]`, which is the only reason you would want to use string field names. It is safe to convert everything to regular camelCase field names.

Otherwise, there is a solution using an enum (like what is done with the `ActionTypes`), but using normal field names is best.

Reply:
While working on it there may be something that is accessed dynamically and needing brackets. the `checkSpace` function in the reducer passes the category as a prop so it might be dynamic:
` if (state.notes[categoryName] !== '') {
           return state.notes[categoryName].concat(' / ', payload);
        }
        else {
            return payload;
        }`.
        
        This can be changed if I pass each comment as separate objects so I'll be working on that